pkg/ovs/ovsctl/appctl.go

```diff
@@ -55,16 +55,18 @@ var knownFeatures = map[DPFeature]struct{}{
 	CTStateNATFeature: {},
 }
 
-type ovsCtlClient struct {
-	bridge string
-	// To allow injection for testing.
-	runAppCtl func(cmd string, needsBridge bool, args ...string) ([]byte, *ExecError)
-}
-
 func NewClient(bridge string) *ovsCtlClient {
```

Comment:
It doesn't make sense to have this function in `appctl.go`.
I really think we need to re-organize `apptcl.go` and `ofctl.go`
Maybe we could have `apptcl.go` for `ovsAppctlRunner`, `ofctl.go` for `ovsOfctlRunner`, and a new `client.go` or `ovsctl.go` file for `ovsCtlClient`. All 3 interfaces can stay in `interface.go`.

Reply:
That means all the functions for ovsCtlClient like Trace , DumpFlows ,etc. will be in ovsctl.go right?

Reply:
Yes